Formula/d/dpic.rb

```diff
@@ -0,0 +1,26 @@
+class Dpic < Formula
+  desc "Implementation of the GNU pic \"little language\""
+  homepage "https://ece.uwaterloo.ca/~aplevich/dpic/"
+  url "https://ece.uwaterloo.ca/~aplevich/dpic/dpic-2024.01.01.tar.gz"
+  sha256 "a69d8f5937bb400f53dd8188bc91c6f90c5fdb94287715fa2d8222b482288243"
+  license "BSD-2-Clause"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    bin.install "dpic"
```

Comment:
Is there no `make install`?

Reply:
Yeah I think the make install does not respect the prefix argument for some reason